tests/wpunit/AccessFunctionsTest.php

```diff
@@ -1,30 +1,90 @@
 <?php
 
-class AccessFunctionsTest extends \Codeception\TestCase\WPTestCase
-{
-
-    public function setUp()
-    {
-        // before
-        parent::setUp();
-
-        // your set up methods here
-    }
-
-    public function tearDown()
-    {
-        // your tear down methods here
-
-        // then
-        parent::tearDown();
-    }
-
-    // tests
-    public function testMe()
-    {
-	    $actual = graphql_format_field_name( 'This is some field name' );
-	    $expected = 'thisIsSomeFieldName';
-	    self::assertEquals( $expected, $actual );
-    }
-
-}
\ No newline at end of file
+class AccessFunctionsTest extends \Codeception\TestCase\WPTestCase {
+
+	public function setUp() {
+		// before
+		parent::setUp();
+
+		// your set up methods here
+	}
+
+	public function tearDown() {
+		// your tear down methods here
+
+		// then
+		parent::tearDown();
+	}
+
+	// tests
+	public function testMe() {
+		$actual   = graphql_format_field_name( 'This is some field name' );
+		$expected = 'thisIsSomeFieldName';
+		self::assertEquals( $expected, $actual );
+	}
+
+	public function testRegisterInputField() {
+
+		/**
+		 * Register Test CPT
+		 */
+		register_post_type( 'test_cpt', [
+			"label"               => __( 'Test CPT', 'wp-graphql' ),
+			"labels"              => [
+				"name"          => __( 'Test CPT', 'wp-graphql' ),
+				"singular_name" => __( 'Test CPT', 'wp-graphql' ),
+			],
+			"description"         => __( 'test-post-type', 'wp-graphql' ),
+			"supports"            => [ 'title' ],
+			"show_in_graphql"     => true,
+			"graphql_single_name" => 'TestCpt',
+			"graphql_plural_name" => 'TestCpts',
+		] );
+
+		/**
+		 * Register a GraphQL Input Field to the connection where args
+		 */
+		register_graphql_field(
```

Comment:
Nice to see it's cleaner and easier to write tests now!

Reply:
💪